frontend/simple/views/SignUp.vue

```diff
@@ -70,14 +71,14 @@ export default {
     submit: function () {
       this.response = ''
       request.post(`${process.env.API_URL}/user/`)
-      .send(serialize(this.form, {hash: true}))
+      .send(serialize(this.form = this.$el.getElementsByTagName('form')[0], {hash: true}))
```

Comment:
We will have to talk about this one. Inititially, I thought it was vue-form setting the this.form attribute. I didn't realize the formhook was also being encompassed by vue-form and the true creator of this attribute. This does not seem needed so I will change the form to a ref and remove the hook.